src/preamble.js

```diff
@@ -1501,5 +1594,48 @@ function lookupSymbol(ptr) { // for a pointer, print out all symbols that resolv
 
 var memoryInitializer = null;
 
-// === Body ===
+#if USE_PTHREADS
```

Comment:
how about if we have a file `src/pthreads_support.js` which we paste in, in `jsifier.js`, if pthreads are used? Just to avoid this and maybe some other big ifdefs that are pthreads-only? i don't feel strongly on this, just wondering if it would be cleaner that way.